src/store/actions/cohorts.actions.js

```diff
@@ -13,3 +13,22 @@ export const fetchCohortsThunk = () => dispatch => {
     .get(API_URL)
     .then(res => dispatch(setCohortsData(res.data.data)));
 };
+
+export const fetchActiveCohort = cohortID => dispatch => {
+  //TODO: add error handling
+  dispatch({ type: SINGLE_COHORT.GET_COHORT_PENDING });
+
+  return axios
+    .get('http://www.mocky.io/v2/5de32f5a3000007200e9c988')
+    .then(res => {
+      const { data } = res;
+      dispatch({
+        type: SINGLE_COHORT.SET_DATA,
+        payload: data[cohortID],
```

Comment:
How come we're looking for data with a key of cohortID here? Based on the backend endpoint we already have set up, I believe we're only returning data for one specific cohort already.

For example, the data from the endpoint will look like:
```
{
data: {
    "gender": [
      { "name": "female", "count": 40 },
      { "name": "male", "count": 4 }
    ], 
   "minorityGroup": [ ... ],
   ...
  }
}
```

I believe the extra "cohort-6" level you see in the mural is for use within the singleCohort part of the store to differentiate between the different cohorts we have stored.

Reply:
I see I thought the data form the endpoint would look like
`{
  "cohort-7": {
    "gender": [
      { "name": "female", "count": 40 },
      { "name": "male", "count": 4 }
    ]
  }
}
`

Reply:
Update. We agreed on this structure:
```{
data: {
    "id":"cohort-7",
    "gender": [
      { "name": "female", "count": 40 },
      { "name": "male", "count": 4 }
    ], 
   "minorityGroup": [ ... ],
   ...
  }
}```